app/api.py

```diff
@@ -107,6 +107,25 @@ def post(self):
         return {'success': True, 'message': msg, 'writeCount': len(indicators)}
 
 
+class EventsManagement(Resource):
+
+    """Perform actions related to events."""
+
+    def get(self):
+        """Get recorded events."""
+        args = parser.parse_args()
+        auth = check_auth(args, role=["analyst", "admin"])
+        if not auth['success']:
+            return auth
+
+        output = {'events': list()}
+        events = [x for x in mongo.db.events.find({}, {'_id': 0})]
```

Comment:
There's a lot of valuable information inside of the event. I think it makes sense to just wrap this events inside of the return dict and carry on. No need to cherry-pick from the event.